app/indices/account_index.rb

```diff
@@ -19,12 +19,7 @@
   has :tenant_id, type: :integer
   has :state, type: :string
 
-  # if System::Database.oracle?
-  #   # Need to add the group by otherwise it will complain about ORA-00979 not a Group By function error
-  #   group_by "accounts.state"
-  # end
-
-  # where sanitize_sql(['COALESCE(accounts.master, ?) = ?', false, false])
+  scope { Account.where(master: false).includes(:users, :bought_cinstances) }
```

Comment:
Why not this?
```suggestion
  scope { Account.not_master.includes(:users, :bought_cinstances) }
```
And if you are sure that master can never be `nil` and it is always `false` instead, then we can also simplify:
https://github.com/3scale/porta/blob/23c541044bf72ea2e9632129877ef0d393e9e32c/app/models/account.rb#L91